src/templates/operations_problem_templates.jl

```diff
@@ -94,7 +102,107 @@ function template_economic_dispatch(; kwargs...)
 
     services = get(kwargs, :services, Dict())
 
-    template = template_unit_commitment(devices = devices, services = services; kwargs...)
+    template = _generic_template(devices = devices, services = services; kwargs...)
 
     return template
 end
+
+"""
+    EconomicDispatchProblem(system::PSY.System; kwargs...)
+
+Creates an `OperationsProblemTemplate` with default DeviceModels for an EconomicDispatch
+problem. Uses the template to create an `OperationsProblem`.
+
+# Example
+```julia
+ed_problem = EconomicDispatchProblem(system)
+```
+
+# Accepted Key Words
+- `network::Type{<:PM.AbstractPowerModel}` : override default network model settings
+- `devices::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
+- `branches::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
+- `services::Dict{Symbol, ServiceModel}` : override default `ServiceModel` settings
+"""
+
+function EconomicDispatchProblem(system::PSY.System; kwargs...)
```

Comment:
Make this a constructor of the struct

Reply:
This will happen when renaming the above struct